### PR TITLE
PBMC and Plasma Manifest updates

### PR DESCRIPTION
**Pull request by roshni-b**
* Expanded the Receiving Party field to cover CIMAC PIs not just the general site
* Added in Inter-site delivery for the Courier field
* Added a Diagnosis Verification field to the Essential Patient Data tab
* Switched the Cohort with Collection Event Name order to place the timepoints next to the IDs
* Updated PBMC and Plasma templates for tests

### CODE REVIEW

File: template_examples/csvs/pbmc_template__Essential_Patient_Data.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 "","Path Concordance Verification","","","","","","ICD-0-3 Code/Description","","","","","Demographics"
-"#h","Entry (#)","Participant id","Cimac id","Surgical pathology report id","Clinical report id","Collection event name","Site description","Topography code","Topography description","Histology/behavior","Histology/behavior description","Sex","Race","Ethnic group"
+"#h","Entry (#)","Participant id","Cimac id","Surgical pathology report id","Clinical report id","Collection event name", "Diagnosis verification", "Site description","Topography code","Topography description","Histology/behavior","Histology/behavior description","Sex","Race","Ethnic group"
```
Comment (curlup) [inline template_examples/csvs/pbmc_template__Essential_Patient_Data.csv]: shouldn't values for this new column also appear in all next rows ?

File: template_examples/csvs/plasma_template__Essential_Patient_Data.csv [hunk 2]
```diff
@@ -1,3 +1,3 @@
 "","Path Concordance Verification","","","","","","ICD-0-3 Code/Description","","","","","Demographics"
-"#h","Entry (#)","Participant id","Cimac id","Surgical pathology report id","Clinical report id","Collection event name","Site description","Topography code","Topography description","Histology/behavior","Histology/behavior description","Sex","Race","Ethnic group"
-"#d","1","TEST-PA01-A1","CM-TEST-PA01-A1","Surgical pathology report 1","clinical report 1","Baseline","ADRENAL GLANDS","C00.2","External lower lip","8003/3","Neoplasm, malignant","Female","Black/African American","Not Hispanic or Latino"
+"#h","Entry (#)","Participant id","Cimac id","Surgical pathology report id","Clinical report id","Collection event name", "Diagnosis verification", "Site description","Topography code","Topography description","Histology/behavior","Histology/behavior description","Sex","Race","Ethnic group"
```
Comment (curlup) [inline template_examples/csvs/plasma_template__Essential_Patient_Data.csv]: same here: shouldn't values for this new column also appear in all next rows ?

File: cidc_schemas/schemas/templates/manifests/plasma_template.json [hunk 3]
```diff
@@ -106,6 +106,10 @@
                         "collection event name": {
                             "merge_pointer": "/collection_event_name",
                             "type_ref": "sample.json#properties/collection_event_name"
+                        },
+                        "diagnosis_verification": {
```
Comment (curlup) [inline cidc_schemas/schemas/templates/manifests/plasma_template.json]: I actually just realized - this would be better without underscore.

Comment (curlup) [reply cidc_schemas/schemas/templates/manifests/plasma_template.json]: Also I somehow don't see this column added to pbmc_template.json. Could you address that too

Comment (roshni-b) [reply cidc_schemas/schemas/templates/manifests/plasma_template.json]: Sure. Addressed in #199

**Pull request merged**